src/sentry/feedback/blueprints/api.md

```diff
@@ -0,0 +1,315 @@
+    - h
+    - d
+    - w
+  - start (optional, string) - ISO 8601 format (`YYYY-MM-DDTHH:mm:ss.sssZ`)
+  - end (optional, string) - ISO 8601 format. Required if `start` is set.
+  - utc (optiona, 'true') - Whether start/end should use the UTC timezone
+  - cursor (optional, str)
+  - per_page (optional, number) Default: 10
+  - offset (optional, number) Default: 0
+  - query (optional, string) - Search query with space-separated field/value
+    pairs. ie: `?query=browser:Firefox`.
+  - queryReferrer(optional, string) - Specify the page which this query is being
+    made from. Used for cross project query on issue replays page. Pass
+    `queryReferrer=replayUserFeedback` for this query.
+
+### Browse Feedback [GET]
+
+Retrieve a collection of feedback items.
+
+**Attributes**
+
+| Column            | Type             | Description                                         |
+| ----------------- | ---------------- | --------------------------------------------------- |
+| browser           | object           | -                                                   |
+| browser.name      | optional[string] | -                                                   |
+| browser.version   | optional[string] | -                                                   |
+| contact_email     | string           | The contact email of the user writing feedback.     |
+| device            | object           | -                                                   |
+| device.brand      | optional[string] | -                                                   |
+| device.family     | optional[string] | -                                                   |
+| device.model      | optional[string] | -                                                   |
+| device.name       | optional[string] | -                                                   |
+| dist              | optional[string] |                                                     |
+| environment       | string           |                                                     |
+| feedback_id       | string           |                                                     |
+| locale            | object           | Browser locale information.                         |
+| locale.lang       | string           | The language preference of the user's browser.      |
+| locale.timezone   | string           | The timezone the feedback was submitted from.       |
+| message           | string           | The message written by the user providing feedback. |
+| os                | object           | -                                                   |
+| os.name           | optional[string] | -                                                   |
+| os.version        | optional[string] | -                                                   |
+| platform          | string           |                                                     |
+| project_id        | number           |                                                     |
```

Comment:
Can we cast this to string on the server?

Reply:
do we return it as a string in other places?

Reply:
We do it on the /replays api.